src/Modules/Flight/src/Flight/Flight.csproj

```diff
@@ -1,9 +1,22 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+	  <OutputType>Exe</OutputType>
```

Comment:
Please remove OutputType tag, it's not necessary.

Reply:
> Please remove OutputType tag, it's not necessary.

I applied the changes and I used top level statement like Api layer for program.cs but 
for top-level statements in C# to work, the project must be configured as an executable. By removing this tag, the project will default to a class library, which does not support top-level statements. 
The <OutputType> tag specifies whether the project is a library or an executable. 
With the <OutputType> set to Exe, our top-level statements in Program.cs will work as intended.

Should I remove the <OutputType> tag or leave it as is?

Reply:
Here in Api layer we don't have any additional configuration and it can host as well. also you need to add Properties folder and add configuration json file for host like Api layer. You need the follow the configuration exactly like Api layer and also add Properties folder and afterward each module should host properly:
[Api.csproj](https://github.com/meysamhadeli/booking-modular-monolith/blob/main/src/Api/src/Api.csproj)

Reply:
I followed your instructions and added a Properties folder with a configuration JSON file to each module, similar to the API layer. I also changed the project structure to Microsoft.NET.Sdk.Web. However, I'm not sure if everything is correct?

Reply:
> I followed your instructions and added a Properties folder with a configuration JSON file to each module, similar to the API layer. I also changed the project structure to Microsoft.NET.Sdk.Web. However, I'm not sure if everything is correct? for launchSetting i changed the port for each module. should all ports be exactly the same as api layer?

Thanks. Port should be different for each module. everything's seems good.